mod/aalborg_theme/views/default/aalborg_theme/css.php

```diff
@@ -73,14 +73,15 @@
 .elgg-icon-hover-menu,
 .elgg-icon-hover-menu:hover,
 :focus > .elgg-icon-hover-menu {
-	background: transparent url(<?= elgg_get_simplecache_url('aalborg_theme/avatar_menu_arrows.png') ?>) no-repeat;
-	background-position: right bottom;
 	width: 100%;
 	height: 100%;
 }
 .elgg-icon-hover-menu-hover:before,
 .elgg-icon-hover-menu:before {
-	content: none;
+	position: absolute;
+	bottom: 0;
+	right: 0;
+	color: #fff;
```

Comment:
do not change to white as that cause visual issues... just leave as default (or switch to aalborgs BLACK color, as used in the image)